apps/explorer/src/components/tx-time/TxTimeType.tsx

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useTimeAgo } from '../../utils/timeUtils';
+
+type Prop = {
+    timestamp: number | undefined;
+};
+
+export function TxTimeType({ timestamp }: Prop) {
```

Comment:
We could also make this a base UI component and port over the time utilities.

Reply:
We can combine this component with the DateCard component on the other PR